src/utils/chartJs/CustomTooltip.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Fade from '@material-ui/core/Grow';
+
+const styles = {
+  tooltip: {
+    background: 'rgba(0, 0, 0, .8)',
+    color: 'white',
+    borderRadius: '4px',
+    pointerEvents: 'none',
+  },
+  tooltipKey: {
+    display: 'inline-block',
+    width: '10px',
+    height: '10px',
+    marginRight: '10px',
+  },
+  table: {
+    borderCollapse: 'collapse',
+  },
+};
+
+function CustomTooltip({ classes, tooltipModel }) {
+  const titleLines = tooltipModel.title || [];
+  const bodyLines = tooltipModel.body
+    ? tooltipModel.body.map(bodyItem => bodyItem.lines)
+    : [];
+  const footerLines = tooltipModel.footer || [];
+
+  return (
+    <Fade in={tooltipModel.opacity === 1}>
+      <div
+        className={classes.tooltip}
+        style={{
+          padding: `${tooltipModel.yPadding}px ${tooltipModel.xPadding}px`,
+        }}
+      >
+        <table className={classes.table}>
+          <thead>
```

Comment:
do not render anything in the <table>`.  You will be able to find the `titleLines` (aka the data of the datapoint somewhere in the `tootipModel` structure.